src/main/java/core/basesyntax/dao/FruitsDao.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.dao;
+
+public interface FruitsDao {
```

Comment:
```suggestion
public interface FruitDao {
```
Don't need to define it in plural